plugins/home/api-report.md

```diff
@@ -44,7 +44,7 @@ export type ClockConfig = {
 export const ComponentAccordion: (props: {
   title: string;
   expanded?: boolean | undefined;
-  Content: () => JSX.Element;
+  Content: () => JSX.Element /** @public */;
```

Comment:
Hmm, I'm not sure why this has been changed, does not seem related 🤔

Reply:
I have a feeling this is related to the build failures as well

Reply:
Fixed